packages/react-components/src/box/src/Box.tsx

```diff
@@ -0,0 +1,46 @@
+import { ComponentProps, ElementType, ForwardedRef, ReactElement, ReactNode } from "react";
+import { forwardRef, omitProps } from "../../shared";
+
+export interface InnerBoxProps {
```

Comment:
I thought all our props interface were supposed to inherit HtmlElement or something more specific like HTMLLinkElement for special cases?